clients/web/src/views/widgets/FileListWidget.js

```diff
@@ -7,6 +7,22 @@ girder.views.FileListWidget = girder.View.extend({
             this.collection.fetchNextPage();
         },
 
+        'click a.g-show-info': function (e) {
+            var cid = $(e.currentTarget).attr('file-cid');
+            new girder.views.FileInfoWidget({
+                el: $('#g-dialog-container'),
+                model: this.collection.get(cid),
+                parentView: this
+            }).on('g:uploadFinished', function () {
+                girder.events.trigger('g:alert', {
```

Comment:
Thanks for finding this, this is cruft left over from copypasta.

To answer your question, that's a global event that can be triggered anywhere in the app, the handler shows a popup with a message in the upper right corner for some period of time.